packages/plugin-console-breadcrumbs/console-breadcrumbs.js

```diff
@@ -9,7 +9,11 @@ exports.init = (client) => {
     console[method] = (...args) => {
       client.leaveBreadcrumb('Console output', reduce(args, (accum, arg, i) => {
         // do the best/simplest stringification of each argument
-        let stringified = String(arg)
+        let stringified = '[Unknown value]'
```

Comment:
Can this be set in the catch block?

Reply:
It can, yeah. What are the advantages of doing so?

Reply:
It just seems more readable to me, rather than having a default and trying to assign, it would be trying to assign and having a fallback, which fits the intended purpose better.